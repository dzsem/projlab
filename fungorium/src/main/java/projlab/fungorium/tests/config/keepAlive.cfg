# Keep Alive
# ------------------------------------------
# Init state
# ------------------------------------------
SET TECTONSPLITCHANCE 0

# 0
ADD TECTON
# 1
ADD KEEPALIVETECTON
REGISTER NEIGHBOUR 0 1
# 2
ADD MUSHROOMBODY 0 0
# 3
ADD MUSHROOMTHREAD 0 0
SET MUSHROOMTHREAD STATE 3 UNCUT GROWN
# 4
ADD MUSHROOMTHREAD 0 1
SET MUSHROOMTHREAD STATE 4 UNCUT GROWN
REGISTER CONNECTION 3 4
# 5
ADD INSECT 1 0