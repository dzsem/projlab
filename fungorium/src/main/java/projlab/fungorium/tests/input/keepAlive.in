# Keep Alive
# ------------------------------------------
# Input
# ------------------------------------------
INSECT 5 CUTTHREAD 3
NEXTROUND
NEXTROUND
NEXTROUND
# Print states
# ------------------------------------------
LIST MUSHROOMTHREAD