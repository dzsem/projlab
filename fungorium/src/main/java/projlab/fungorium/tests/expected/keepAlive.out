MUSHROOMTHREAD 3 2 0 0 CUT GROWN
MUSHROOMTHREAD 4 -1 1 0 CUT GROWN